app/javascript/vue/SpeedRunsLiveRaceList.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+  <div class="card">
+    <div class="card-header justify-content-between d-flex">
+      <span>
+        <img src="http://cdn.speedrunslive.com/images/srl_305.png" style="height: 1em" />
+        <small
+          class="fas fa-info-circle"
+          v-tippy
+          title="For your convenience of not needing to check two platforms, this is a list of open races on SpeedRunsLive. Splits.io is not affiliated with SpeedRunsLive."
+        ></small>
+      </span>
+      <span class="text-danger text-uppercase" v-tippy title='This list updates live; refreshing is not required.'>Live</span>
+    </div>
+    <div class="list-group bg-transparent" v-cloak>
+      <span class="spinner-border spinner-border-sm mx-auto m-5" role="status" v-if="loading">
+        <span class="sr-only">Loading...</span>
+      </span>
+      <speed-runs-live-race v-for="(race, i) in races" v-model="races[i]" />
```

Comment:
Does this need the `key` directive to make replacing them work properly?  I remember having issues with this on the streams component when it wasn't set it would just remove the first or last entry instead of the relevant one.